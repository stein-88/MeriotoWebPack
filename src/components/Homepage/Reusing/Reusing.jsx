import React from 'react'
import img1 from '@images/homeimg1.svg'
import img2 from '@images/homeimg2.png'
import img3 from '@images/homeimg3.png'
import schema from './data'

const Reusing = () => {
    const { titulo } = schema
    return (
        <div className="row">
            <div className="col-12">
                <h2 className="h1 mt-5 py-4 text-muted">{titulo}</h2>
                <section className="pt-4 text-center w-100">
                    <img className="img-fluid" src={img1} alt="companies" />
                    <img className="img-fluid" src={img2} alt="companies" />
                    <img className="img-fluid" src={img3} alt="companies" />
                </section>
            </div>
        </div>
    )
}

export default Reusing
