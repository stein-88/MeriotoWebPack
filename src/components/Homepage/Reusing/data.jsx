const schema = {
    titulo: 'Get Reusing',
}

export default schema
