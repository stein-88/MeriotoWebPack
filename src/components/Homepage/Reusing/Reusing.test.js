import React from 'react'
import renderer from 'react-test-renderer'
import Reusing from './Reusing'

describe('Testing Reusing component', () => {
    const compReusing = renderer.create(<Reusing />)
    test('Should render without props', () => {
        const jsonReusing = compReusing.toJSON()
        expect(jsonReusing).toMatchSnapshot()
    })
    test('Should have 3 images and 1 title', () => {
        const resultIMG = compReusing.root.findAllByProps({ className: 'img-fluid' })
        const resultTitle = compReusing.root.findAllByProps({ className: 'h1 mt-5 py-4 text-muted' })
        expect(resultIMG).toHaveLength(3)
        expect(resultTitle).toHaveLength(1)
    })
})
