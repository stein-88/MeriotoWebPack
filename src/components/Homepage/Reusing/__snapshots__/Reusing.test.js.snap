// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Reusing component Should render without props 1`] = `
<div
  className="row"
>
  <div
    className="col-12"
  >
    <h2
      className="h1 mt-5 py-4 text-muted"
    >
      Get Reusing
    </h2>
    <section
      className="pt-4 text-center w-100"
    >
      <img
        alt="companies"
        className="img-fluid"
        src="test-file-stub"
      />
      <img
        alt="companies"
        className="img-fluid"
        src="test-file-stub"
      />
      <img
        alt="companies"
        className="img-fluid"
        src="test-file-stub"
      />
    </section>
  </div>
</div>
`;
